docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -1,23 +1,55 @@
-# Styling UI components with XAML
+# Styling UI components
 Styling of components in NUI is possible in two ways:
-  - `ViewStyle`: designed for simple themes.
-  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
+  - `ViewStyle`: a set of attributes to decorate a `View`.
+  - `Theme`: a set of `ViewStyle` to decorate an application. You can load the theme from a XAML resource.
```

Comment:
Couldn't see this comment updated: 
https://github.com/Samsung/tizen-docs/pull/1441#discussion_r640042748

https://docs1.stg.tizen.org/staging/1441/application/dotnet/guides/nui/styling-controls-with-xaml/
Actually, the occurrence is at two places, that's why you probably missed it. 
can you please verify?

Reply:
Done.